### PR TITLE
WIP-map2: mercator map component

### PR DESCRIPTION
**Pull request by kimj42**
Moved functions that displayed the map in debug console in the DebugConsole.jsx from WIP-map1 to a component on its own. Wanted to get feedback on whether it looks good as is.

Ignore d3-scale for now as it should be used later on as I start to plot lat/long on the map.

Related WIP-map1 PR:https://github.com/code-dot-org/code-dot-org/pull/29100

### CODE REVIEW

File: apps/src/lib/tools/jsdebugger/DebugConsole.jsx [hunk 1]
```diff
@@ -77,6 +79,7 @@ const UNWATCH_COMMAND_PREFIX = '$unwatch ';
 // The JS console, by default, prints 'undefined' when there is no return value.
 // We don't want that functionality and therefore don't include 'undefined' in this list.
 const FALSY_VALUES = new Set([false, null, 0, '', NaN]);
+
```
Comment (epeach) [inline apps/src/lib/tools/jsdebugger/DebugConsole.jsx:82]: Nit: remove added line.

File: apps/src/lib/tools/jsdebugger/MercatorMap.jsx [hunk 2]
```diff
@@ -0,0 +1,68 @@
+// import PropTypes from 'prop-types';
```
Comment (epeach) [inline apps/src/lib/tools/jsdebugger/MercatorMap.jsx:1]: Nit: remove commented line

File: apps/src/lib/tools/jsdebugger/MercatorMap.jsx [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+import React from 'react';
+// import {connect} from 'react-redux';
```
Comment (epeach) [inline apps/src/lib/tools/jsdebugger/MercatorMap.jsx:3]: Nit: remove commented line

File: apps/src/lib/tools/jsdebugger/MercatorMap.jsx [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+import {geoMercator, geoPath} from 'd3-geo';
+import {feature} from 'topojson-client';
+
+const RADIUS = 20;
+const WIDTH = 400;
+const HEIGHT = 400;
+
+class MercatorMap extends React.Component {
+  // static propTypes = {
```
Comment (epeach) [inline apps/src/lib/tools/jsdebugger/MercatorMap.jsx:12]: Nit: remove commented lines

File: apps/src/lib/tools/jsdebugger/MercatorMap.jsx [hunk 5]
```diff
@@ -0,0 +1,68 @@
+// import PropTypes from 'prop-types';
+import React from 'react';
+// import {connect} from 'react-redux';
+import {geoMercator, geoPath} from 'd3-geo';
+import {feature} from 'topojson-client';
+
+const RADIUS = 20;
```
Comment (epeach) [inline apps/src/lib/tools/jsdebugger/MercatorMap.jsx:7]: Can we update these variable names to describe what the radius/width/height are for? (ex: "MERCATOR_MAP_WIDTH")

**Review by epeach — COMMENTED** [review]
Looks good! Just a few nits to address.
Can you add a screenshot or gif to the pull request description?

File: apps/src/lib/tools/jsdebugger/MercatorMap.jsx (hunk 5 above)
Comment (kimj42) [reply apps/src/lib/tools/jsdebugger/MercatorMap.jsx:7]: Yes, I'll change it the variable names to be more descriptive.

**Comment by kimj42** [comment]
There is now a final version of this PR so will close this WIP PR.

**Pull request closed without merging**